### DISCUSSION TITLE
Where should I send my scans to?

### DISCUSSION THREAD

**Discussion by falbalae** [papermerge/papermerge-core#352] [Q&A]
HelloI
I would like to send my scans directly to a folder which papermerge is continuosly monitoring. As for the moment being I have no idea which would be the correct path to send the scans to. Anyone has an idea? 

Currently I have to scan the document on my NAS first and then manually move it into papermerge. I would like this to be automatic. 

Thanks.

**Comment by ciur** [comment]
How do you "send" your scans? Or how do you plan to "send"? Via REST API?

**Reply by falbalae** [reply]
Hello @ciur and thanks for answering!

I am sorry but I have no knowledge in programming: I am just a user.

My scanner sends the scanned documents to a network folder which is stored on my Synology NAS. The ideal situation would be for Papermerge to monitor such folder and automatically archive those scanned documents in the right folder or to add the right tag (I am connecting to the topic I was mentioning in [discussion #351](https://github.com/papermerge/papermerge-core/discussions/351)).

Papermerge is installed on the NAS. 

Thanks for any advice! :-)
